spec/models/profile/profile_identifiers_spec.rb

```diff
@@ -14,7 +14,7 @@
       },
       {
         value: 'XYZ123456',
-        identifier_type: :prison_number,
+        identifier_type: :criminal_records_office,
```

Comment:
changing a test like this looks wrong - you are possibly removing a test for CRO?